docs: wear os template (#710)
VeryGoodOpenSource/very_good_cli: site/docs/templates/wear.md

---
sidebar_position: 8
---

# Flutter Wear OS app 🤖⌚️

> Template of a Wear OS a based on the [Core][core_link]. It includes VGV best practices with the addition of specific setup to make Flutter Apps stand out on Wear OS devices.

Reviewer: ```suggestion
Template for a Wear OS app based on the [Core][core_link]. It includes VGV best practices with additional setup to make Flutter apps stand out on Wear OS devices.
```

![Very Good Wear_App][wear_os_photo]

:::note
Read more about this app template [in our blog][blog].
:::

## App Features ✨

- **Condensed Layout** - The layout is optimized for the small screen size of Wear OS devices.

> - **Hooked into the hardware** - The app has a sample usage of the hardware resources of the device.

Reviewer: ```suggestion
- **Hooked into the Hardware** - The app has a sample usage of the hardware resources of the device.
```
